Gems/Atom/Feature/Common/Assets/Shaders/SkyBox/MultiViewSkyBox.shader

```diff
@@ -0,0 +1,26 @@
+{
+    "Source" : "SkyBox.azsl",
+        
+    "Definitions": ["QUALITY_LOW_END_TIER1=1", "QUALITY_LOW_END_TIER2=1"],
```

Comment:
I feel like these QUALITY_LOW_END flags are for material shaders specifically. I would prefer to just set ENABLE_PHYSICAL_SKY and ENABLE_MERGE_FILMIC_TONEMAP here directly.

Reply:
ENABLE_MERGE_FILMIC_TONEMAP is used for material shaders too. I will pull out ENABLE_PHYSICAL_SKY.